tests/test_io.py

```diff
@@ -18,6 +18,9 @@ def test_getVariables():
     varB = getTokens("xy+ xy^2 +yz^3")
     assert getVariables(varB) == ['x', 'y', 'z']
 
+    varC = getTokens("x + sin(x) + cos(y) + tan(2*z)*2 + tanh(z) + e^2")
```

Comment:
Comment out this section. 
```suggestion
    # FIXME: getVariables() in visma.io.checks
    # varC = getTokens("x + sin(x) + cos(y) + tan(2*z)*2 + tanh(z) + e^2")
    # assert getVariables(varC) == ['x', 'y', 'z']
```